src/main/reactjs/src/interfaces/clerkTranslatorEmail.ts

```diff
@@ -0,0 +1,22 @@
+import { Action } from 'redux';
+
+import { APIResponseStatus } from 'enums/api';
+import { AppRoutes } from 'enums/app';
+
+export interface ClerkTranslatorEmail {
+  subject: string;
+  body: string;
+}
+
+export interface ClerkTranslatorEmailState {
+  status: APIResponseStatus;
+  email: ClerkTranslatorEmail;
+  recipients: Array<number>;
+  redirect?: AppRoutes;
+}
+
+export interface ClerkTranslatorEmailAction extends Action<string> {
+  email: Partial<ClerkTranslatorEmail>;
+  recipientIds: Array<number>;
+  redirect?: AppRoutes;
```

Comment:
Tarvitaanko redirectejä enää, jos se tapahtuu on success tilassa aina automaattisesti?

Reply:
Tarvitaan redirect kuitenkin cancel-dialogiin.

Reply:
Joko omana kenttänä redux-tilassa, tai sitten jos haluaa laajentaa `APIResponseStatus` enumia esim `cancelled`-tilalla tms.

Reply:
Näitä tapaksia tulisi varmaan jatkossa lisää,  canceled tila kuulostaa aika hyvältä mun korviin :)

Reply:
Muutettu.